package.json

```diff
@@ -19,7 +19,8 @@
   "author": "Krystian Kościelniak",
   "license": "MIT",
   "dependencies": {
-    "is-subset": "^0.1.1"
+    "lodash.difference": "^4.5.0",
```

Comment:
generally looks good, thanks =D

however it would be nice if you:
- wrote a regression test
- upgrade the NPM version of this library